elk/evaluation/evaluate_log.py

```diff
@@ -0,0 +1,27 @@
+from dataclasses import dataclass
+
+from elk.training.reporter import EvalResult
+
+
+@dataclass
+class EvalLog:
+    """The result of running eval on a layer of a dataset"""
+
+    layer: int
+    eval_result: EvalResult
+
+    @staticmethod
+    def csv_columns() -> list[str]:
+        return ["layer", "acc", "cal_acc", "auroc", "ece"]
+
+    def to_csv_line(self) -> list[str]:
+        items = [
+            self.layer,
+            self.eval_result.acc,
+            self.eval_result.cal_acc,
+            self.eval_result.auroc,
+            self.eval_result.ece,
```

Comment:
previously the columns were ["layer", "loss", "acc", "cal_acc", "auroc"] for whatever Eval logged. So i think was a bug where acc was written to to "loss" instead, and "cal_acc" to "acc" instead, etc.